lib/plugins/bed.js

```diff
@@ -84,7 +84,7 @@ function inject (bot) {
 
   function sleep (bedBlock, cb) {
     cb = cb || noop
-    if (!(bot.time.day >= 12541 && bot.time.day <= 23458)) {
+    if (!(bot.time.timeOfDay >= 12541 && bot.time.timeOfDay <= 23458)) {
```

Comment:
What about defining isDay to timeOfDay out of [12542, 23458] so it can be used here ?

Reply:
I guess that just depends on when you consider it to be night.
That definition would be the time at which you can sleep.
I used the moment mobs are able to spawn.
https://minecraft.gamepedia.com/Daylight_cycle

Reply:
ok, fair enough.